luci/luci-app-ffwizard-falter/po/en/ffwizard-falter.po

```diff
@@ -78,9 +78,9 @@ msgstr ""
 "highest bandwidth. Therefor we need to know, how much bandwidth to the "
 "internet there is and how much do you want to share. Typical values are 6.0 "
 "Mbit/s download and 0.5 Mbit/s upload with <em>DSL 6000</em> or 50.0 Mbit/s "
-"download and 10.0 Mbit/s upload with <em>VDSL 50000</em>. Preferably, you "
+"download and 10.0 Mbit/s upload with <em>VDSL 50000</em>. Preferable, you "
```

Comment:
Oops, I made a mistake.  It should be "Preferably".  Just get rid of the ","  That was my intention last time.

Reply:
Adjusted in the new commit.